pennylane/ops/op_math/sum.py

```diff
@@ -525,8 +525,9 @@ def compute_grouping(self, grouping_type="qwc", method="lf"):
         _, ops = self.terms()
 
         with qml.QueuingManager.stop_recording():
-            op_groups = qml.pauli.group_observables(ops, grouping_type=grouping_type, method=method)
-        self._grouping_indices = tuple(tuple(ops.index(o) for o in group) for group in op_groups)
+            self._grouping_indices = qml.pauli.compute_partition_indices(
```

Comment:
Was this causing a test to fail?

Reply:
Also note that there's a technical roadmap task to make this change.

Reply:
> Was this causing a test to fail?
No, I just stumbled upon it and it felt like a straightforward enough change.